bundles/org.openhab.binding.mqtt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -94,6 +94,44 @@
 				<advanced>true</advanced>
 			</parameter>
 
+			<parameter name="birthMessage" type="text">
+				<label>Birth Message</label>
+				<description>The message to send to the broker when a connection is established.</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="birthTopic" type="text">
+				<label>Birth Topic</label>
+				<description>Defaults to empty and therefore disables the birth message.</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="birthRetain" type="boolean">
+				<label>Birth Message Retain</label>
+				<description>True if the birth message should be retained (defaults to true)</description>
+				<default>true</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="shutdownMessage" type="text">
+				<label>Shutdown Message</label>
+				<description>The message to send to the broker before the connection terminates.</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="shutdownTopic" type="text">
+				<label>Shutdown Topic</label>
+				<description>Defaults to empty and therefore disables the shutdown message.</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="shutdownRetain" type="boolean">
+				<label>Shutdown Message Retain</label>
+				<description>True if the shutdown message should be retained (defaults to true)</description>
+				<default>false</default>
+				<advanced>true</advanced>
+			</parameter>
```

Comment:
Please add these texts resources/OH-INF/i18n/mqttbroker.properties also - either manually or by running `mvn i18n:generate-default-translations`.

Reply:
`mvn i18n:generate-default-translations` doesn't work for me.

[ERROR] No plugin found for prefix 'i18n' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (~/.m2/repository), central (https://repo1.maven.org/maven2), openhab-release (https://openhab.jfrog.io/openhab/libs-release)] -> [Help 1]

Reply:
This command worked for me:

```
mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations
```

Reply:
Hmm, there's already a mqtt**broker**.properties file. Can you check what's up with that? I just checked that previous parameter strings (compared to your changes) existed in that file, but don't know why it's named like that and if it is actually correctly found and used. At least there seems to be a conflict.

Reply:
Good catch! I just ran the command above. Either the command I used is outdated, or the other files used an outdated naming scheme? Either way, we'd need to pick one or the other. What would you suggest?

Reply:
I think I understand this now that I've had a proper look into it. The name should be mqtt.properties as per what was just generated by the i18n tool. I'll push a fix shortly to include the other translations.

Reply:
So the other file never had any effect? Could be tested by simply adding two different strings to one of the strings in each file so it would be visible which file is actually used.

Reply:
@jlaur, I will push the properties fix in a separate PR so that it is noted in the change log. The existing translations aren't taking effect currently.

Reply:
The files are renamed in #12193